src/servers/src/postgres/handler.rs

```diff
@@ -244,6 +260,16 @@ impl ExtendedQueryHandler for PostgresServerHandler {
 
         let sql_plan = &portal.statement.statement;
 
+        if sql_plan.query.is_empty() {
+            // early return if query is empty
+            return Ok(Response::EmptyQuery);
+        }
+
+        if let Some(mut resps) = fixtures::process(&sql_plan.query, query_ctx.clone()) {
```

Comment:
Since fixtures are rare cases, should we use `matches` first to short-circuit if it's normal SQL?

Reply:
I think `process` is just `matches` and execute, it makes no difference if it doesn't match. Otherwise we might need to match twice for those actually in our fixture whitelist.